web-common/src/features/dashboards/workspace/DashboardHeader.svelte

```diff
@@ -49,18 +63,18 @@
     title and call to action
   -->
   <div
-    style:height="var(--header-height)"
     class="flex items-center justify-between w-full pl-1 pr-4"
+    style:height="var(--header-height)"
   >
     <!-- title element -->
     <h1 style:line-height="1.1" style:margin-top="-1px">
       <div class="pl-4" style:font-family="InterDisplay" style:font-size="20px">
         {displayName || metricViewName}
       </div>
     </h1>
-    <!-- top right CTAs -->
-    {#if isEditableDashboard}
-      <div style="flex-shrink: 0;">
+    <PanelCTA>
```

Comment:
This triggers a `svelte-check` error: "Property 'side' is missing in type '{}' but required in type '{ side: \"left\" | \"right\"; }'."